compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -91,6 +92,8 @@ class t_java_generator : public t_oop_generator {
         reuse_objects_ = true;
       } else if( iter->first.compare("option_type") == 0) {
         use_option_type_ = true;
+      } else if( iter->first.compare("handle_runtime_exceptions") == 0) {
```

Comment:
Yep, should be added to the structure at the end of the file at the very least.

It would be nice to have more comprehensive documentation on the java options for the compiler in the java runtime readme, perhaps... but getting it into the compiler's --help output is required as stated above.